CHANGES.md

```diff
@@ -84,6 +84,8 @@
 
 <!-- For example, Docker, GitHub Actions, pre-commit, editors -->
 
+- Github Actions: adds 2 variables as github action outputs to report changes that can
+  be used in other steps. (#3531)
```

Comment:
```suggestion
- GitHub Action: adds 2 action outputs to report changes that can
  be used in other steps. (#3531)
```
Please also mention the output names in this changelog entry.

Reply:
0 problems, I'll make the requested changes ..

Reply:
Regarding re-emitting the stderr - I have a feeling that black uses some mechanism to check if its output is actual console or just a file buffer and it will strip the terminal escape sequences based on the target. This will happen with straight piping and also when piped thru  `tee`

Reply:
- is_formatted & changed_files renamed to use - instead of _
- outputs mentioned in changelog and docs updated to reflect the rename
- changed is-formatted to have str like bool.